src/pages/HomePage.tsx

```
@@ -0,0 +1,165 @@
+          {/* Hero Section */}
+          <div className="text-center mb-12">
+            <h1 className="text-4xl md:text-6xl font-bold mb-4">Margea</h1>
+            <p className="text-lg md:text-xl text-base-content/70 max-w-2xl mx-auto">
+              Encontre e agrupe Pull Requests do Renovate Bot de forma inteligente.
+            </p>
+          </div>
+
+          {/* Login prompt if not authenticated */}
+          {!isAuthenticated && (
+            <div className="alert alert-info mb-8">
+              <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" className="stroke-current shrink-0 w-6 h-6">
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)